dpdata/vasp/outcar.py

```diff
@@ -49,8 +49,18 @@ def get_outcar_block(fp) :
             return blk
     return blk
 
+def get_outcar_block_ml(fp) :
+    blk = []
+    for ii in fp :
+        if not ii :
+            return blk
+        blk.append(ii.rstrip('\n'))
+        if 'free  energy ML TOTEN' in ii:
+            return blk
+    return blk
+
 # we assume that the force is printed ...
-def get_frames (fname, begin = 0, step = 1) :
+def get_frames (fname, begin = 0, step = 1,ml = False) :
```

Comment:
This line does not follow [PEP-8](https://peps.python.org/pep-0008/).

Reply:
add whitespace after the comma